src/com/google/enterprise/adaptor/filenet/Permissions.java

```diff
@@ -39,131 +39,108 @@
       AccessRight.READ_AS_INT | AccessRight.VIEW_CONTENT_AS_INT;
   private static final int USE_MARKING = AccessRight.USE_MARKING_AS_INT;
 
-  private final AccessPermissionList perms;
-  private final String owner;
-
-  public Permissions(AccessPermissionList perms, String owner) {
```

Comment:
We should add a TODO to implement owners in ACLs (which is easy since we copy the whole ACL chain for each document), and I would vote for preserving the constructor and call rather than deleting them now and resurrecting them shortly.